.changeset/eighty-dolls-smile.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-kubernetes-common': patch
```

Comment:
I noticed you mapped another error code [here](https://github.com/backstage/backstage/pull/14714/files#diff-13d169acd3ba469c379c517e548b237cef164b02bd066e85b0b31cefd222e844R181). Is the newly introduced error type related to the main change or something additional you're doing in the same pull request? If the changes are unrelated, could you please create a separate changeset file for the "@backstage/plugin-kubernetes-common" changes?

Reply:
Done https://github.com/backstage/backstage/pull/14714/commits/8aab6a3922dcb6b1afb0c6d91d9c67ea6c00fd12